sklearnex/_config.py

```diff
@@ -63,6 +75,11 @@ def set_config(target_offload=None, allow_fallback_to_host=None, **sklearn_confi
         If True, allows to fallback computation to host device
         in case particular estimator does not support the selected one.
         Global default: False.
+    compute_mode : str or list, default=None
+        The computational method used for BLAS level 3 routines in
+        oneDAL for select GPU devices. The compute_mode string(s)
+        must be from the oneMKL compute_mode enum list.
```

Comment:
`compute_mode` is too broad term for config parameter. `mkl_blas_compute_mode` might be better.